kombu/transport/redis.py

```diff
@@ -1066,14 +1066,17 @@ class SentinelChannel(Channel):
     from_transport_options = Channel.from_transport_options + (
         'master_name',
         'min_other_sentinels',
-        'sentinel_kwargs')
+        'sentinel_kwargs',
+        'connection_kwargs')
 
     connection_class = sentinel.SentinelManagedConnection if sentinel else None
 
     def _sentinel_managed_pool(self, async=False):
         connparams = self._connparams(async)
 
-        additional_params = connparams.copy()
+        additional_params = getattr(self,
+                                    'connection_kwargs',
+                                    connparams.copy())
```

Comment:
Perhaps I didn't explain my suggestion very well. So, I meant that perhaps the two dictionaries should be merged, since they both contain different but overlapping sets of parameters:

```
additional_params = connparams.copy()
connection_kwargs = getattr(self, 'connection_kwargs', {})
additional_params.update(connection_kwargs)
```

Reply:
so what's the resolution ?